scico/test/linop/test_binop.py

```diff
@@ -0,0 +1,48 @@
+import operator as op
+
+import pytest
+
+import scico.numpy as snp
+from scico import linop
+
+
+class TestBinaryOp:
+    def setup_method(self, method):
+        self.input_shape = (5,)
+        self.input_dtype = snp.float32
+
+    @pytest.mark.parametrize("operator", [op.add, op.sub])
+    def test_case1(self, operator):
+        A = linop.Convolve(
+            snp.ones((2,)), input_shape=self.input_shape, input_dtype=self.input_dtype, mode="same"
+        )
+        B = linop.Identity(input_shape=self.input_shape, input_dtype=self.input_dtype)
+
+        assert type(A + B) == linop.LinearOperator
+        assert type(B + A) == linop.LinearOperator
+        assert type(2.0 * A + 3.0 * B) == linop.LinearOperator
+        assert type(2.0 * B + 3.0 * A) == linop.LinearOperator
```

Comment:
parameterized by `operator` but `operator` is never used?

Reply:
Good catch. Fixed.